Resources/Prototypes/Research/industrial.yml

```diff
@@ -200,15 +200,16 @@
   - ClothingBackpackDuffelHolding
 
 - type: technology
-  id: PortableFission
-  name: research-technology-portable-fission
+  id: MaximizedPowercells
+  name: research-technology-maximized-powercells
```

Comment:
i require this name to be at least 75% saucier. you won't take my sweet boy "portable fission" and replace it with "maximized powercells"

tech should sound cool